src/app/child-dev-project/notes/model/note.ts

```diff
@@ -166,6 +183,25 @@ export class Note extends Entity {
     }
 
     this.children = this.children.concat(childId);
+    this.relatedEntities.push(
```

Comment:
What is the benefit of this partial migration of children and schools (but not users and recurring activities)? Woudln't it be easiert to make a full migration at one point so we don't have to worry about what has already been migrated and what not?